src/EditorFeatures/Core.Wpf/QuickInfo/OnTheFlyDocsView.xaml.cs

```diff
@@ -137,6 +139,14 @@ private async Task SetResultTextAsync(ICopilotCodeAnalysisService copilotService
 
         try
         {
+            //if (_onTheFlyDocsElement.IsContentExcluded)
```

Comment:
Remove the comment. Also we are no longer logging  telemetry when symbol is excluded, just  want to make sure its intended